employees/templates/employees/report_list.html

```diff
@@ -105,6 +105,9 @@ <h4><font color="red">{{ error }}</font></h4>
 {% endblock %}
 
 {% block extra_script %}
+    {{ form.media }}
+    <script src="//code.jquery.com/jquery-3.3.1.min.js"></script>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/jquery.mask/1.14.10/jquery.mask.js"></script>
```

Comment:
Looks a little bit suspicious that we use two versions of the same library, maybe removing older version below will keep everything still working?

Reply:
good point. changed.